docs/puml/showScheduleForADate.puml

```diff
@@ -0,0 +1,73 @@
+@startuml
+
+hide footbox
+
+participant User as User
+participant Main as Main
```

Comment:
The diagram is hard to read, I suggest using ref and extract the interaction between Timetable and TaskList.